### PR TITLE
Proposed changes to index.md related to mkdocs #818 & #824

### PR DESCRIPTION
**Pull request by brianjking**
Proposed changes to index.md related to mkdocs #818 & #824

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ S3, or anywhere else you choose.
 ### Great themes available
 
 There's a stack of good looking themes included by default. Choose from
-bootstrap, readthedocs, or any of the 12 bootswatch themes.
+bootstrap, readthedocs, or any of the 12 bootswatch themes. You can also check out a list of 3rd party themes in the [MkDocs wiki](https://github.com/mkdocs/mkdocs/wiki/MkDocs-Themes), or better yet, add your own.
```
Comment (waylan) [inline docs/index.md]: Your getting a Markdown-Lint [failure][1] here for line length.

[1]: https://travis-ci.org/mkdocs/mkdocs/jobs/110445743#L196

Comment (brianjking) [inline docs/index.md]: @waylan Oops, sorry! Pushed a fix.

**Comment by d0ugal** [comment]
Looks good, there is a new linting error with training whitespace but I'll fix that.

**Comment by d0ugal** [comment]
If you are curious, fixed in https://github.com/mkdocs/mkdocs/commit/f7525626b88ad53c035c07f343374fda4a40ad00

**Pull request merged**